online/package.json

```diff
@@ -3,31 +3,24 @@
   "private": true,
   "version": "0.0.0",
   "scripts": {
-    "res:build": "rescript build",
-    "res:clean": "rescript clean",
-    "res": "rescript build -w",
     "vendor": "rsync -av --exclude='node_modules' --exclude='lib' --exclude='coverage' --exclude='__tests__' --exclude='.git' --exclude='online' ../ ./vendor/rescript-json-schema",
     "dev": "vite",
```

Comment:
Is the compiler built-in in the Vite plugin?

Reply:
Yes